Readme updates to introduction technologies for GraphQL (#3433)
redwoodjs/redwood: README.md

- Forms with easy client- and/or server-side validation and error handling.
- [Hot module replacement](https://webpack.js.org/concepts/hot-module-replacement/) (HMR) for faster development.
- Database migrations (via Prisma 2).

> - [Envelop Plugins](https://www.envelop.dev) that enhances the GraphQL lifecycle from context to execution

Reviewer: ```suggestion
- [Envelop Plugins](https://www.envelop.dev) that enhance the GraphQL lifecycle from context to execution
```

> - Simple, but powerful GraphQL Directives to validate access or transform resolved data

Reviewer: ```suggestion
- Simple but powerful GraphQL Directives to validate access or transform resolved data
```

- Logging using [Pino](https://getpino.io) including to [transports](https://getpino.io/#/docs/transports)
- Signature verification and payload signing for handling incoming and outgoing Webhooks
- Page prerendering
- First class JAMstack-style deployment to [Netlify](https://www.netlify.com/), [Vercel](https://vercel.com/), [Render](https://render.com/), and [Serverless](https://www.serverless.com/).

## Roadmap

> We intend to ship v1 in the fourth quarter of 2021.

Reviewer: ```suggestion
We intend to have a 1.0 release candidate before the end of 2021 and to release a truly production-ready 1.0 in early 2022.
```
